client/src/homepage.js

```diff
@@ -17,6 +18,24 @@ export class Homepage extends React.Component {
             <Link to="/en-US/docs/Web/HTML/Element/canvas">HTML/canvas</Link>
           </li>
         </ul>
+        <h3>HTML Guide Pages</h3>
+        <ul>
+          <li>
+            <Link to="/en-US/docs/Web/HTML/Element">
+              Index of HTML elements
```

Comment:
Technically this one's a landing page not a guide page, but I don't think this is important.